### PR TITLE
BaSP-TG-34: Select and input components

### PR DESCRIPTION
**Pull request by FedeBrus64**
This components work the following way:
- They receive the following properties: id, required, value; to set them on the html part of the components (Input also receives the type of the data introduced, i.e. text, number, etc).
- There is also a text property, to set the text that the label will show before the input/select.

This components have been implemented and seem to work. Regardless of that, make sure to let me know if something is not working as it should.

### CODE REVIEW

**Review by Julian25 — APPROVED** [review]
Excellente job!!

**Review by CorinaArreguez — APPROVED** [review]

File: src/Components/Shared/Input/Input.jsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    <div className={styles.inputContainer}>
+      <label className={styles.labelInput}>{text}</label>
+      <input
+        className={styles.inputBox}
+        type={type}
+        id={id}
+        required={required}
+        value={value}
+        onChange={onChange}
+      ></input>
```
Comment (BaltazarCortina1) [inline src/Components/Shared/Input/Input.jsx:15]: La etiqueta de cierre no es necesaria. Podes poner `/>` en la misma etiqueta de apertura

File: src/Components/Shared/Input/Input.jsx [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import React from 'react';
```
Comment (BaltazarCortina1) [inline src/Components/Shared/Input/Input.jsx:1]: Cambiar los nombres de los archivos a index

File: src/Components/Shared/Select/Select.jsx [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+      <label className={styles.labelSelect}>{text}</label>
+      <select
+        className={styles.selectBox}
+        id={id}
+        value={value}
+        required={required}
+        onChange={onChange}
+        type={type}
+      >
+        {options}
```
Comment (BaltazarCortina1) [inline src/Components/Shared/Select/Select.jsx:16]: Aca falta hacer los cambios que hablamos en la pre-entrega para adaptarlo al componente form

File: src/Components/Shared/Modal/Modal.jsx [hunk 4]
```diff
@@ -0,0 +1,43 @@
+import React from 'react';
```
Comment (BaltazarCortina1) [inline src/Components/Shared/Modal/Modal.jsx:1]: Elimina el componente Modal de esta branch asi podemos mergearlo por separado

**Review by BaltazarCortina1 — CHANGES_REQUESTED** [review]

**Review by lautaro23 — APPROVED** [review]

**Review by BaltazarCortina1 — APPROVED** [review]

**Pull request merged**